models/topics.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('..//db/connection')
+
+exports.selectTopics = () => {
+    return db.query(
+        `SELECT * FROM topics`
+    ).then(({rows}) => {
```

Comment:
Nice destructuring. Remember this is the job of the model to prepare the data for the controller to send on so continue this pattern going forwards 👍